http_service/pulse/consumers.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
Nit: since the amount of code is relatively limited, could you merge the two files?